Examples/Examples/Models/Examples.swift

```diff
@@ -150,6 +150,9 @@ public struct Examples {
                 type: LineGradientExample.self),
         Example(title: "Animate a GeoJSON line",
                 description: "Update a map's style to animate a line over time.",
-                type: AnimateGeoJSONLineExample.self)
+                type: AnimateGeoJSONLineExample.self),
+        Example(title: "Use a map & annotations with SwiftUI",
+                description: "Use the UIViewRepresentable protocol to wrap a MapView in a SwiftUI view",
```

Comment:
Think CI will complain that this needs a full-stop at the end. Can you run the Examples scheme for testing?